LICENSE.md

```diff
@@ -0,0 +1,6 @@
+Tracim is distributed under the terms of 4 distinct licenses:
+
+- AGPLv3 for the Agenda application (`frontend_app_agenda` folder). The text of this license can be found in [LICENSE_AGPLv3](LICENSE_AGPLv3).
+- LGPL for other frontend parts (`frontend` and `frontend_xxx` folders). The text of this license can be found in [LICENSE_LGPLv3](LICENSE_LGPLv3).
+- CC by SA for the provided welcome page background images (These are all the images whose path starts with `frontend/dist/assets/branding/welcome-simple-bg` in this repository). They are Copyright (c) Guénael Muller. The text of this license can be found in [LICENSE_CC_BY_SA](LICENSE_CC_BY_SA).
```

Comment:
```suggestion
- CC-BY-SA for the provided welcome page background images (These are all the images whose path starts with `frontend/dist/assets/branding/welcome-simple-bg` in this repository). They are Copyright (c) Guénael Muller. The text of this license can be found in [LICENSE_CC_BY_SA](LICENSE_CC_BY_SA).
```
You should also give the version of the license.  @inkhey, which version? 4.0 or later?
https://creativecommons.org/licenses/by-sa/4.0/

Reply:
actually the original text was : "Their licence is the Creative Commons Public License Attribution-ShareAlike 3.0 Unported."